app/src/main/res/layout-land/question_player_fragment.xml

```diff
@@ -148,19 +148,20 @@
 
     <FrameLayout
       android:layout_width="match_parent"
-      android:layout_height="wrap_content"
+      android:layout_height="124dp"
```

Comment:
How did you come up with this height?

Reply:
I doubled the height of the textview banner and added the textview's bottom margin.
I think the more appropriate way to set up the layout would be to set the KonfettiView to be 124dp and then setting the FrameLayout to wrap_content, so I'll make that change

Reply:
Done